bip-0325.mediawiki

```diff
@@ -28,7 +28,7 @@ A new type of network ("signet"), which takes an additional consensus parameter
 
 The witness commitment of the coinbase transaction is extended to include a secondary commitment (the signature/solution) of either:
 
-    1-4 bytes - Push the following (4 + x + y) bytes
+    1-5 bytes - Push the following (4 + x + y) bytes
     4 bytes - Signet scriptSig header (0xecc7daa2)
```

Comment:
```suggestion
    4 bytes - Signet header (0xecc7daa2)
```

Could fixup this one as well when touching the adjacent line?

Reply:
Done!